asyncscope.md

```diff
@@ -438,9 +449,7 @@ int main() {
 
 ## Starting parallel work
 
-In this example we use the `counting_scope` within lexical scope to construct an algorithm that performs parallel work.
-This uses the `let_value_with` [@letvwthunifex] algorithm implemented in [@libunifex] which simplifies in-place
-construction of a non-moveable object in the `let_value_with` algorithm's _`operation-state`_ object. Here `foo`
+In this example we use `let_value_with_async_scope` to construct an algorithm that performs parallel work. Here `foo`
```

Comment:
We need to pick between `let_value_with_async_scope` and `let_with_async_scope` (I prefer the latter because it's shorter; I wonder what @anthonywilliams thinks).
```suggestion
In this example we use `let_with_async_scope` to construct an algorithm that performs parallel work. Here `foo`
```